### ISSUE TITLE
[BUG]: Compiler shows error with wrong line

### ISSUE THREAD

**Issue by soraros** [modularml/mojo#1612]
### Bug description

As title. It seems to have something to do with `print`.

### Steps to reproduce

```mojo
# comment

def main():
    print("a", object())
```

The error message reads:

```
# comment
^
/workspaces/ubuntu/issues/print_object.mojo:1:1: note: candidate not viable: callee expects 1 argument, but 2 were specified
# comment
^
/workspaces/ubuntu/issues/print_object.mojo:1:1: note: candidate not viable: callee expects 1 argument, but 2 were specified
# comment
^
/workspaces/ubuntu/issues/print_object.mojo:1:1: note: candidate not viable: callee expects 2 input parameters, but 0 were specified
# comment
^
/workspaces/ubuntu/issues/print_object.mojo:1:1: note: candidate not viable: callee expects 2 input parameters, but 0 were specified (5 more notes omitted.)
# comment
^
/home/vscode/.modular/pkg/packages.modular.com_mojo/bin/mojo: error: failed to parse the provided Mojo
```

### System information

```shell
Mojo 0.6.1 on Docker, Intel Mac
```

**Comment by soraros** [comment]
This seems to be fixed on 0.7.0

**Comment by laszlokindrat** [comment]
Thanks for checking @soraros, please keep filing tickets if you run into issues.
